CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## master / unreleased
 
 * [CHANGE] Querier / ruler: Change `-querier.max-fetched-chunks-per-query` configuration to limit to maximum number of chunks that can be fetched in a single query. The number of chunks fetched by ingesters AND long-term storare combined should not exceed the value configured on `-querier.max-fetched-chunks-per-query`. #4260
+* [FEATURE] Add configuration options in order to register an instance's IPv6 address in the ring instead of IPv4. IPv4 still remains the default option. #4339
+  * New flags added:
+    * `-compactor.instance-addr-protocol`
+    * `-store-gateway.sharding-ring.instance-addr-protocol`
+    * `-ingester.lifecycler.protocol`
```

Comment:
It was named that way because other configs have a variable named *InstanceAddr*, but in /pkg/ring/lifecycler.go it is named *Addr* as you can see [here](https://github.com/cortexproject/cortex/blob/bee49bb15c5ba332d81f34e87408942f7cc74bc3/pkg/ring/lifecycler.go#L63).

It seemed like the proper name for this variable here, but would you rather see it changed?